connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/executable/ActiveExecutableResponse.java

```diff
@@ -29,4 +30,5 @@ public record ActiveExecutableResponse(
     Class<? extends InboundConnectorExecutable> executableClass,
     List<InboundConnectorElement> elements,
     Health health,
-    Collection<Activity> logs) {}
+    Collection<Activity> logs,
+    Long activationTimestamp) {}
```

Comment:
Curious, is activationTimestamp different from the last Health.UP? We could also add a timestamp to health to store the data of state change. Just an idea